src/Microsoft.Sbom.Api/README.md

```diff
@@ -24,20 +42,60 @@
                 Verbosity = System.Diagnostics.Tracing.EventLevel.Verbose,
             };
 
-            await generator.GenerateSBOMAsync(scanPath, scanPath, metadata, null, configuration, outputPath);
+            await Task.Run(async () =>
```

Comment:
Is `Task.Run` necessary? The general advice is that it's only a good practice for CPU-bound code[^1]

[^1]: https://blog.stephencleary.com/2013/10/taskrun-etiquette-and-proper-usage.html

Reply:
You're right it's not necessary. Fixed.